### PR TITLE
S2 add hiring partner page

### PR DESCRIPTION
**Pull request by patrickkenyon**
- **addHiringPartner.phtml** & **addHiringPartner.css** - Created add hiring partner page 
- **routes.php** - Added a new route for page 
- **admin.phtml** & **admin.css** - added nav button to addPartner route and adjusted styling
- **registerUser.phtml** - adjusted indentation from previous cohort because it was rubbish

### CODE REVIEW

File: public/css/addHiringPartner.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.buttonWrapper {
+    margin: 5px 0;
+}
```
[the remark this answers is not in the document]
Comment (patrickkenyon) [reply public/css/addHiringPartner.css]: Changed in 823e766

File: templates/addHiringPartner.phtml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+</head>
+<body>
+    <div>
+        <a href="/admin" class="returnButton">Return to admin page</a>
+    </div>
+    <h1> Add Hiring Partner Page</h1>
+    <div id="message"></div>
+    <form id="addHiringPartnerForm">
+        <h4>Company name</h4>
+        <input type="text" class="addHiringPartner" name="companyName" required>
```
[the remark this answers is not in the document]
Comment (patrickkenyon) [reply templates/addHiringPartner.phtml]: Changed in c6c65e9

File: templates/addHiringPartner.phtml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        <h4>Phone Number</h4>
+        <input type="tel" class="addHiringPartner" name="telNumber">
+        <h4>Website</h4>
+        <input type="url" class="addHiringPartner" name="companyURL">
+        <div class="buttonWrapper">
+            <input type="submit" id="addHiringPartnerSubmit">
+        </div>
+    </form>
+</body>
+</html>
```
[the remark this answers is not in the document]
Comment (patrickkenyon) [reply templates/addHiringPartner.phtml]: Changed in c6e0cec

File: templates/addHiringPartner.phtml [hunk 4]
```diff
@@ -1,42 +1,42 @@
...
+        <div>
+            <a href="/admin" class="returnButton">Return to admin page</a>
         </div>
-    </form>
-</body>
+        <h1> Add Hiring Partner Page</h1>
+        <div id="message"></div>
+        <form id="addHiringPartnerForm">
+            <label for="companyName" class="hiringPartnerLabels">Company name</label>
+            <input type="text" class="addHiringPartner" name="companyName" required>
```
Comment (ucanfil) [inline templates/addHiringPartner.phtml]: label 'for' attribute references the 'id' attribute of the corresponding input so you need to add id='companyName' to your inputs.

Comment (patrickkenyon) [reply templates/addHiringPartner.phtml]: Crap - sorry! Completely forgot. ab705f4

**Pull request merged**